app/controllers/courses_controller.rb

```diff
@@ -10,7 +10,13 @@ def index
                      .per(Settings.courses_per_page)
   end
 
-  def show; end
+  def show
+    @course = Course.find_by id: params[:id]
+    @subjects = @course.subjects.page(params[:page]).per(Settings.subject_per_page)
```

Comment:
nếu course bị nil thì line này có vấn đề j ko em ơi

Reply:
Dạ course bị nil thì line báo ko tồn tại khóa học. còn line 15 thì khi course đó chưa add subject nào vô hết thì vẫn vào được course và trong chi tiết course đó chưa có dữ liệu.

Reply:
> course nil thì hiện flash thông báo, line 15 này em chưa biết cách chỗ xử lí bắt cái course nào chưa được add subject thì thông báo chưa có subject trong course

chưa biết thì hỏi nha chú, thấy bạn làm mà ko có hỏi gì hết trơn á, toàn bị comment mấy cái cơ bản ko à